libraries/AC_AutoTune/AC_AutoTune_Heli.cpp

```diff
@@ -811,7 +811,22 @@ void AC_AutoTune_Heli::dwell_test_run(sweep_info &test_data)
     }
 
     if (settle_time == 0) {
-        target_angle_cd = -chirp_input.update((now - dwell_start_time_ms) * 0.001, degrees(tgt_attitude) * 100.0f);
+        dwell_freq = chirp_input.get_frequency_rads();
+        float tgt_att_limited = tgt_attitude;
+        if (is_positive(dwell_freq)) {
+            float tgt_att_temp = tgt_attitude;
```

Comment:
Why do we need the extra variable?

Reply:
@lthall I think it is needed to hold the rate limited attitude.  If the rate limit is not set then it will skip setting tgt_att_temp variable.  I guess I could just use the tgt_att_limited variable throughout and keep setting it from the MIN function.